<!--  This file consumes information on image_ns/image_raw and depth_ns/image_raw to and publishes results from the detection_server.py backend -->
<launch>
<!-- Uncomment this block to add the ability to auto-launch a docker environment with the required detection backend
        otherwise launch a detection backend rosrun rasberry_perception detection_server.py -backend "backend_name" -backend_arg1 "config.yourconfig"
    <arg name="backend" default=""/>
    <group if="$(eval arg('backend') != '')">
        <node pkg="rasberry_perception" name="rasberry_perception_detection_backend" type="run_backend.sh" args="$(arg backend)" output="screen" required="true"/>
    </group>
-->

<<<<<<< HEAD
    <arg name="image_ns" default="/sequence_0/colour"/>
    <arg name="depth_ns" default="/sequence_0/depth"/>
    <arg name="score" default="0.5"/>

    <node pkg="rasberry_perception" name="rasberry_perception_client" type="detection_client.py" output="screen" respawn="true">
=======
    <arg name="image_ns" default="/camera/color"/>
    <arg name="depth_ns" default="/camera/aligned_depth_to_color"/>
    <arg name="score" default="0.5"/>

    <node pkg="rasberry_perception" name="rasberry_perception_detector" type="detection_client.py" output="screen" respawn="true">
>>>>>>> 5de2510... refactor: Rework containerisation of detections backends
        <param name="image_ns" type="str" value="$(arg image_ns)"/>
        <param name="depth_ns" type="str" value="$(arg depth_ns)"/>
        <param name="score" type="double" value="$(arg score)"/>
    </node>
</launch>
